docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -12,6 +12,23 @@ There you can find links to upgrade notes for other versions too.
     - once you finish this change, you still should deal with older redis cache keys that don't use new prefixes. Such keys are not removed even by `clean-redis-old`, please find and remove them manually (via console or UI)
 
     **Be careful, this upgrade will remove sessions**
-
+- in order to have translations extracted even from overwritten templates update your `build-dev.xml` file accordingly ([#931](https://github.com/shopsys/shopsys/pull/931)):
```

Comment:
It is definitely not low priority. You need to extract translations from your extended templates - period.

Reply:
low priority should mean that the instruction could be done as last one and do not have so high priority. But all instructions are mandatory to upgrade custom project into minor version of project-base.

this one is questionable because there was no change in framework for this PR, but the translations should be extracted in the first step so i agree with @TomasLudvik 
but now from his approach i see one missing instruction to run the updated phing target.

Reply:
@boris-brtan, I do not think, that it should be part of UPGRADE instructions. Developers should know when they want to extract their translations and they might not want to do that during upgrade.

Reply:
hmm, so i will not be sure whether updated phing target will pass... why should i do such update then?

Reply:
@boris-brtan you may want to run that any you may not, that is why I do not want to add it as upgrade instruction

Reply:
OK, make sense.

Reply:
Ok it makes sense that this doesn't need to be low priority